control-plane/grpc/src/operations/node/traits.rs

```diff
@@ -30,6 +33,10 @@ pub trait NodeOperations: Send + Sync {
         get_blockdevice: &dyn GetBlockDeviceInfo,
         ctx: Option<Context>,
     ) -> Result<BlockDevices, ReplyError>;
+    /// Cordon the node with the given ID and associate the label with the cordon.
```

Comment:
` associate the label with the cordon.` --> ` associate the label with the cordoned node.` ?